lib/MusicBrainz/Server/Edit/Relationship/EditLinkType.pm

```diff
@@ -34,7 +34,7 @@ sub change_fields
         description         => Nullable[Str],
         priority            => Optional[Int],
         attributes          => Optional[ArrayRef[Dict[
-            name => Optional[Str], # Used in old historic edits
+            name => Str,
             min  => Nullable[Int],
             max  => Nullable[Int],
             type => Optional[Int], # Used in the new edits
```

Comment:
Should we also add type ID to the old edits for consistency?

Reply:
Actually, I'm not seeing a single edit without a type. Maybe they were added in at some point? I can remove the `Optional` + comment though.

Reply:
If that's the case, then it makes sense I guess :)